app/kuma-dp/pkg/dataplane/envoy/envoy.go

```diff
@@ -162,12 +164,26 @@ func (e *Envoy) Start(stop <-chan struct{}) error {
 		// and we don't expect users to do "hot restart" manually.
 		// so, let's turn it off to simplify getting started experience.
 		"--disable-hot-restart",
-		"-l ", e.opts.LogLevel.String(),
+		"--log-level", e.opts.LogLevel.String(),
 	}
 	if version != "" { // version is always send by Kuma CP, but we check empty for backwards compatibility reasons (new Kuma DP connects to old Kuma CP)
 		args = append(args, "--bootstrap-version", string(version))
 	}
 
+	// If the concurrency is explicit, use that. On Linux, users
+	// can also implicitly set concurrency using cpusets.
+	if e.opts.Config.DataplaneRuntime.Concurrency > 0 {
+		args = append(args,
+			"--concurrency",
+			strconv.FormatUint(uint64(e.opts.Config.DataplaneRuntime.Concurrency), 10),
+		)
+	} else if runtime.GOOS == "linux" {
+		// The `--cpuset-threads` flag is still present on
+		// non-Linux, but emits a warning that we might as well
+		// avoid.
+		args = append(args, "--cpuset-threads")
```

Comment:
I'm wondering why this is not enabled by default in Envoy...

Reply:
https://github.com/envoyproxy/envoy/pull/5975#issuecomment-464114942

Reply:
thanks!